### DISCUSSION TITLE
Location of the Community Custom Cards

### DISCUSSION THREAD

**Discussion by sildehoop** [UI-Lovelace-Minimalist/UI#166] [General]
What are your thoughts on the place of the community custom cards, another folder https://github.com/UI-Lovelace-Minimalist/UI/discussions/158#discussioncomment-1876467 or another repo https://github.com/UI-Lovelace-Minimalist/UI/discussions/105#discussioncomment-1875835?

I think a different repo like https://github.com/dwainscheeren/dwains-dashboard-addons is a good idea this way we can also make releases separate like https://github.com/UI-Lovelace-Minimalist/UI/discussions/105#discussioncomment-1875835? said

**Comment by CM000n** [comment]
In general I like the idea.
I don't know much about the environment of Dwain's dashboard. But as the name suggests, there is probably mainly one person behind the dashboard and the associated decisions.
I find that a little more difficult with us. At least since @Paddy0174  left, there is no longer "the one" person behind the UI. So all of our content is actually community driven and kind of "custom".
If we want to make such a separation, we would have to, in my opinion, first determine together what core components should be.

**Reply by mp-se** [reply]
Since I raised the topic, my thoughts behind the suggestion was to see if we could enable more people to share their design ideas and suggestions for new cards in a custom repository (or several). The main repository would then contain the "standard" cards that are well tested and comply with the overall design of the UI. I'm not a UI designer (more backend programmer) so I would appreciate someone that is better at for instance CSS help out and make my suggestions work better. In a separate custom repo we could perhaps be less strict on what is submitted.

**Reply by CM000n** [reply]
My question remains. What is "standard"? ;-)

**Reply by mp-se** [reply]
My defintion was "cards that are well tested and comply with the overall design of the UI"

**Reply by CM000n** [reply]
Today I thought of something else that might also need to be considered in the course of this. With the HACS support perhaps the handling with Custom Cards and their language files could be more difficult. If these are scattered over different folders and possibly even over their own repo, this makes it more difficult to deal with the respective translation files. Of course, it would be easier if they all shared a common language file.

**Reply by CM000n** [reply]
When I look at Dwain's repo with the dashboard addons, I see no references to files with translations. Maybe the addons share the translation file of the main repo, or do without translations completely?
